output/output.go

```diff
@@ -16,16 +20,16 @@ const (
 // log formats.
 func ParseFormat(format string) Format {
 	switch format {
-	case string(FormatJSON),
+	case string(FormatJSONGCP):
+		return FormatJSONGCP
+	case string(FormatJSON), "logfmt":
 		// True 'logfmt' has significant limitations around certain field types:
 		// https://github.com/jsternberg/zap-logfmt#limitations so since it implies a
 		// desire for a somewhat structured format, we interpret it as OutputJSON.
-		"logfmt":
 		return FormatJSON
-	case string(FormatConsole),
+	case string(FormatConsole), "condensed":
 		// The previous 'condensed' format is optimized for local dev, so it serves the
 		// same purpose as OutputConsole
-		"condensed":
```

Comment:
nit - keep the old formatting, or bring the docstring above the case

Reply:
Oh you're right, while it took me 2m to parse the original syntax with the line break and I like it much better that way, I missed that it messes how the comment reads now. Fixing this!